bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -1572,14 +1572,20 @@ lnd/clm2/surfdata_map/release-clm5.0.30/surfdata_ne0np4.CONUS.ne30x8_hist_78pfts
 <light_res    use_cn=".true." phys="clm4_5"        >94x192</light_res>
 <light_res    use_cn=".true." phys="clm5_0"        >94x192</light_res>
 <light_res    use_cn=".true." phys="clm5_1"        >360x720</light_res>
+<light_res    use_fates=".true." phys="clm4_5"     >360x720</light_res>
+<light_res    use_fates=".true." phys="clm5_0"     >360x720</light_res>
 
 <stream_year_first_lightng use_cn=".true."   >0001</stream_year_first_lightng>
 <stream_year_last_lightng  use_cn=".true."   >0001</stream_year_last_lightng>
+<stream_year_first_lightng use_fates=".true.">0001</stream_year_first_lightng>
+<stream_year_last_lightng  use_fates=".true.">0001</stream_year_last_lightng>
 
 <stream_fldfilename_lightng hgrid="94x192"    use_cn=".true."   >atm/datm7/NASA_LIS/clmforc.Li_2012_climo1995-2011.T62.lnfm_Total_c140423.nc</stream_fldfilename_lightng>
 <stream_fldfilename_lightng hgrid="360x720"   use_cn=".true."   >atm/datm7/NASA_LIS/clmforc.Li_2016_climo1995-2013.360x720.lnfm_Total_c160825.nc</stream_fldfilename_lightng>
+<stream_fldfilename_lightng use_fates=".true."   >atm/datm7/NASA_LIS/clmforc.Li_2016_climo1995-2013.360x720.lnfm_Total_c160825.nc</stream_fldfilename_lightng>
```

Comment:
Roger that, I think I see what you're saying.  So it would be:
`<stream_fldfilename_lightng hgrid="360x720"   use_fates=".true.  >atm/datm7/NASA_LIS/clmforc.Li_2016_climo1995-2013.360x720.lnfm_Total_c160825.nc</stream_fldfilename_lightng>"`

Reply:
Updated via https://github.com/ESCOMP/CTSM/pull/1264/commits/677f422c98de1d4b5548df020a86ee13828d3e45.